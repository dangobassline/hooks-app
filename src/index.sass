h1
    font-family: 'Staatliches', cursive