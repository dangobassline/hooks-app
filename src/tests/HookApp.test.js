import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import { HookApp } from "../../src/HookApp";


describe('Test al componente HookApp', () => {

    test('should hacer el Snapshot del componente correctamente', () => {
        const wrapper = shallow( <HookApp /> );
        expect(wrapper).toMatchSnapshot();
    });

});
