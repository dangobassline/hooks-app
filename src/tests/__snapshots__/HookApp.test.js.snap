// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test al componente HookApp should hacer el Snapshot del componente correctamente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
